tests/security/selinux/selinux_setup.pm

```diff
@@ -60,14 +60,27 @@ sub run {
     # for opensuse, e.g, Tumbleweed install selinux_policy pkgs as needed
     # for sle15 and sle15+ "selinux-policy-*" pkgs will not be released
     # NOTE: have to install "selinux-policy-minimum-*" pkg due to this bug: bsc#1108949
-    if (!is_sle && !is_leap || is_sle('>=15')) {
+    # install policy packages separately due to this bug: bsc#1177675
+    if (is_sle('>=15')) {
         my @files
           = ("selinux-policy-20200219-3.6.noarch.rpm", "selinux-policy-minimum-20200219-3.6.noarch.rpm", "selinux-policy-devel-20200219-3.20.noarch.rpm");
         foreach my $file (@files) {
             assert_script_run "wget --quiet " . data_url("selinux/$file");
             assert_script_run("rpm -ivh --nosignature --nodeps --noplugins $file");
         }
     }
+    elsif (!is_sle && !is_leap) {
+        my @files
+          = ("selinux-policy-20201016-1.1.noarch.rpm", "selinux-policy-minimum-20201016-1.1.noarch.rpm", "selinux-policy-devel-20201016-1.1.noarch.rpm");
+        foreach my $file (@files) {
+            assert_script_run "wget --quiet " . data_url("selinux/$file");
+            assert_script_run("rpm -ivh --nosignature --nodeps --noplugins $file");
+        }
+    }
+
+    # record the pkgs' version for reference
+    my $results = script_output("zypper se -s selinux-policy");
+    record_info("FYI", "SELinux policy packages' version is: $results");
```

Comment:
This is a good idea to show the package version. :-) 
Just my personal idea about It looks you can change the name as an example: **Pkg_ver** or something else instead of "FYI", how do you think?  It's Up to you.

Reply:
> **Pkg_ver**

Done.